### PR TITLE
Add wicket and tez (newly detected)

### PR DESCRIPTION
**Pull request by MyEnthusiastic**

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -2935,6 +2935,9 @@ https://github.com/Apache/Struts,13d9053050c9e4fb2ef049db6a37d3f6eebf48fa,plugin
 https://github.com/Apache/Struts,13d9053050c9e4fb2ef049db6a37d3f6eebf48fa,plugins/portlet,org.apache.struts2.portlet.dispatcher.Jsr286DispatcherTest.testRender_ok,OD-Vic,Accepted,https://github.com/apache/struts/pull/263,
 https://github.com/Apache/Struts,13d9053050c9e4fb2ef049db6a37d3f6eebf48fa,plugins/rest,org.apache.struts2.rest.handler.JacksonXmlHandlerTest.testObjectToXml,ID,Accepted,https://github.com/apache/struts/pull/500,
 https://github.com/Apache/Struts,3a7c35a014f8a3471a1b75e7546e01ecf21d5db3,core,org.apache.struts2.views.freemarker.FreemarkerResultMockedTest.testDynamicAttributesSupport,ID,Accepted,https://github.com/apache/struts/pull/506,
+https://github.com/apache/tez,51d6f53967110e2b91b6d90b46f8e16bdc062089,tez-api,org.apache.tez.dag.api.TestDAG.testRecreateDAG,ID,Opened,https://github.com/apache/tez/pull/316,
+https://github.com/apache/tez,51d6f53967110e2b91b6d90b46f8e16bdc062090,tez-api,org.apache.tez.dag.api.TestDAGPlan.userVertexOrderingIsMaintained,ID,Opened,https://github.com/apache/tez/pull/317,
```
Comment (zzjas) [inline pr-data.csv:2939]: Was this PR opened by your account? I saw a different account.

File: pr-data.csv [hunk 2]
```diff
@@ -2935,6 +2935,9 @@ https://github.com/Apache/Struts,13d9053050c9e4fb2ef049db6a37d3f6eebf48fa,plugin
...
+https://github.com/apache/tez,51d6f53967110e2b91b6d90b46f8e16bdc062091,tez-dag,org.apache.tez.dag.app.dag.impl.TestVertexImpl.testVertexInit,ID,Opened,https://github.com/apache/tez/pull/318,
```
Comment (zzjas) [inline pr-data.csv:2940]: This repo doesn't have `pull/318` now. Was this a typo?

**Comment by MyEnthusiastic** [comment]
Updaed, all the repos are in "add tried"

**Comment by darko-marinov** [comment]
This PR is failing the checker. If you want us to update this before the other, make more changes here so that the checker passes.

**Comment by darko-marinov** [comment]
The CI is failing.

**Comment by MyEnthusiastic** [comment]
@zzjas @darko-marinov  Hey, this PR has been fixed 😊

**Comment by zzjas** [comment]
Great!

Ok to merge @darko-marinov

**Comment by darko-marinov** [comment]
You just need to resolve conflicts here and get many "points" rather than lose -2.

**Comment by MyEnthusiastic** [comment]
OH... Done

**Pull request merged**